gptel-org.el

```diff
@@ -288,14 +288,13 @@ for inclusion into the user prompt for the gptel request."
 (defun gptel-org--link-standalone-p (object)
   "Check if link OBJECT is on a line by itself."
   ;; Specify ancestor TYPES as list (#245)
-  (let ((par (org-element-lineage object '(paragraph))))
+  (when-let* ((par (org-element-lineage object '(paragraph))))
     (and (= (gptel-org--element-begin object)
             (save-excursion
-              (goto-char (org-element-property :contents-begin par))
+              (goto-char (and par (org-element-property :contents-begin par)))
```

Comment:
This will never be reached if `par` is nil, because of the `when-let*`.  So all you really need is the original code but with the `let*` replaced by a `when-let*`.

If you're not sure how this works, try macroexpanding the `when-let*` clause.